publication-commons/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     implementation libs.nva.auth
     implementation libs.nva.secrets
     implementation libs.nva.clients
+    implementation libs.guava
```

Comment:
We had a transitive dependency on guava and piggybacked off this, it seems, I now express this here…it's a shame as the usage is a bit pointless